taxcalc/functions.py

```diff
@@ -293,22 +298,21 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900,
               c19700 + c20500 + c20800 + e21000 + e21010)
 
     # Limitations on deductions excluding medical, charity etc
-    _phase2_i = ID_ps[MARS - 1]
-
     _nonlimited = c17000 + c20500 + e19570 + e21010 + e20900
     _phase2_i = ID_ps[MARS - 1]
     _limitratio = _phase2_i / _sep
 
+    # Itemized deductions amount after limitation if any
+    c04470 = c21060
```

Comment:
Form 1040 Schedule A: Line 29. Corresponding to case:
"No. Your deduction is not limited. Add the amounts in the far right column for lines 4 through 28. Also, enter this amount on Form 1040, line 40."